site/docs/guides/operations/kubernetes/operator.md

```diff
@@ -108,7 +108,7 @@ Kubernetes is a useful container solution that scales from a raspberry pi to hug
 
 #### Egeria Platform
 
-The [OMAG server platform](https://odpi.github.io/introduction/overview/#omag-server-platform) (aka 'server chassis') is effectively a java process that is launched and acts as a container for
+The [OMAG server platform](/concepts/omag-server-platform) (aka 'server chassis') is effectively a java process that is launched and acts as a container for
```

Comment:
Why is this link not fully qualified but the others are?

Reply:
@mandy-chessell mentioned that these should be internal links on the previous pull request - see: https://github.com/odpi/egeria-docs/pull/339#discussion_r812077858

Reply:
ok cool